server/game/Events/EntersPlayEvent.js

```diff
@@ -1,11 +1,13 @@
 const Event = require('./Event.js');
 
 class EntersPlayEvent extends Event {
-    constructor(params, card, fate, gameAction) {
+    constructor(params, card, fate, status, gameAction) {
         super('onCharacterEntersPlay', params);
         this.handler = this.entersPlay;
         this.card = card;
         this.fate = fate;
+        this.card.isHonored = status === 'honored';
+        this.card.isDishonored = status === 'dishonored';
```

Comment:
We can't change the card state in the constructor, because if something cancels this event, we'll end up with an honored card in hand/province.  Changing the state needs to be in the handler.  I also think it might be better to honor/dishonor by calling the honor/dishonor method on the card.

Reply:
Good catch, I suppose I should be at least doing it in the same place the `card.fate` is changed down below. I'll try calling it in via the `dishonor` method on the card but I did that briefly yesterday and it didn't work (might have just been PEBKAC).